### PR TITLE
presentation role conflict resolution tests

### LINKED ISSUE #1938: wpt tests for presentational conflict resolution
Create tests for 
https://w3c.github.io/aria/#conflict_resolution_presentation_none

### PR DESCRIPTION
**Pull request by jnurthen**
resolves https://github.com/w3c/aria/issues/1938

### CODE REVIEW

File: wai-aria/role/role_none_conflict_resolution.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!-- none with label(global) on header -->
+<h1 role="none" data-testname="none with global" data-expectedrole="heading" aria-label="x" class="ex">x</h1>
```
Comment (cookiecrook) [inline wai-aria/role/role_none_conflict_resolution.html:15]: What does "none with global" mean as the test name? "any tagname with role none and with any global attribute"? If so, can we name the test more explicitly like: 
```suggestion
<h1 role="none" data-testname="heading role none with global attr aria-label" data-expectedrole="heading" aria-label="x" class="ex">x</h1>
```

File: wai-aria/role/role_none_conflict_resolution.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<!-- none with label(global) on paragraph -->
+<p role="none" data-testname="none with global (prohibited role)" data-expectedrole="paragraph" aria-label="x" class="ex">x</p>
+
+<!-- none with focusable header -->
+<h1 role="none" data-testname="none with focusable 0" data-expectedrole="heading" tabindex="0" class="ex">x</h1>
+<h1 role="none" data-testname="none with focusable -1" data-expectedrole="heading" tabindex="-1" class="ex">x</h1>
```
Comment (cookiecrook) [inline wai-aria/role/role_none_conflict_resolution.html:22]:
```suggestion
<h1 role="none" data-testname="focusable heading role none with tabindex=0" data-expectedrole="heading" tabindex="0" class="ex">x</h1>
<h1 role="none" data-testname="focusable heading role none with tabindex=0" data-expectedrole="heading" tabindex="-1" class="ex">x</h1>
<h1 role="none" data-testname="non-focusable heading role none" data-expectedrole="none" class="ex">x</h1>
```

**Review by cookiecrook — CHANGES_REQUESTED** [review]
Mostly making test names more specific, since those are effectively the "permalinks" of WPT.fyi

File: wai-aria/role/role_none_conflict_resolution.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<!-- none with non-global-->
+<h1 role="none" data-testname="none with non-global" data-expectedrole="none" class="ex" aria-level="2"> Sample Content </h1>
+
```
[the remark this answers is not in the document]
Comment (jnurthen) [reply wai-aria/role/role_none_conflict_resolution.html:28]: Not arguing the need for these but I don't think they fit in this test

[the remark this answers is not in the document]
Comment (cookiecrook) [reply wai-aria/role/role_none_conflict_resolution.html:30]: okay to disregard then. Thanks.

**Review by cookiecrook — APPROVED** [review]

File: wai-aria/role/role_none_conflict_resolution.html [hunk 4]
```diff
@@ -22,7 +22,7 @@ <h1 role="none" data-testname="heading role none with global attr aria-label" da
 
 <!-- none with focusable header -->
 <h1 role="none" data-testname="focusable heading role none with tabindex=0" data-expectedrole="heading" tabindex="0" class="ex">x</h1>
-<h1 role="none" data-testname="focusable heading role none with tabindex=0" data-expectedrole="heading" tabindex="-1" class="ex">x</h1>
+<h1 role="none" data-testname="focusable heading role none with tabindex=-1" data-expectedrole="heading" tabindex="-1" class="ex">x</h1>
```
Comment (cookiecrook) [inline wai-aria/role/role_none_conflict_resolution.html:25]: Sorry about that. Good catch.

**Pull request merged**